### ISSUE TITLE
Can ParseArguments let through unparsed arguments?

### ISSUE THREAD

**Issue by timmkrause** [commandlineparser/commandline#536]
The call to my CLI looks like this:

`some-cli foo --some:configuration=something ...`

I am using `ParseArguments()` in the following way (`MapResult` omitted here):

`Parser.Default.ParseArguments<FooOption, BarOption>(args);`

The command line returns:

```
ERROR(S):
  Option 'some:configuration' is unknown.
```

The options look identical, here is a sample of `FooOption`:

```csharp
[Verb("foo", HelpText = "Some helpful text.")]
internal class FooOption
{
}
```

Is there a way to handle additional, unparsed and unmapped arguments in a more flexible way? Meaning to ignore them?

In other words: Map whatever fits and let everything else pass.

**Comment by moh-hassan** [comment]
Can you post the code of classes `FooOption, BarOption ` to know the format of the arguments.?

**Comment by timmkrause** [comment]
@moh-hassan I added it to the question.

**Comment by moh-hassan** [comment]
Your `FooOption` class is empty without some options.
It can be:
```cs
[Verb("foo", HelpText = "Some helpful text.")]
 internal class FooOption
 {
    [Option()]
    public string Some {get;set;}
 }
```
The commandline arguments can be(excluding app name):

```
//<verb><--option><=><value>        
   equal sign is next to option_name without space
   foo --some=configuration:something

OR
// <verb><--option><space><value>  
 one space or more  is next to option_name 
foo --some configuration:something
```    

FooOption sfter parsing :
```
FooOption.Some = "configuration:something"
```

>Can ParseArguments let through unparsed arguments

yes, configure parser with [IgnoreUnknownArguments ](https://github.com/commandlineparser/commandline/wiki/P_CommandLine_ParserSettings_IgnoreUnknownArguments)=true:

```cs

var parser = new CommandLine.Parser (with => {
		with.IgnoreUnknownArguments = true;   //ignore unknown options without firing errors.
		//other options
	});
 	parser.ParseArguments<FooOption,BarOption> (args)
       //....  		
```

For parser option details:, see [ParserSettings Class](https://github.com/commandlineparser/commandline/wiki/T_CommandLine_ParserSettings#properties)

**Comment by timmkrause** [comment]
I still want to make use of

```csharp
new ConfigurationBuilder()
    // ...
    .AddCommandLine(args)
    // ...
```

for certain configuration arguments (`Microsoft.Extensions.Configuration`) and be more flexible that way.

So for that need

```csharp
new CommandLine.Parser(
    with => {
		with.IgnoreUnknownArguments = true;
    });
```

is perfect for my use case.

Thank you!
